fe/fe-core/src/main/java/com/starrocks/staros/StarOSBDBJEJournalSystem.java

```diff
@@ -205,7 +208,7 @@ private boolean replayJournal(JournalCursor cursor)
                 break;
             }
 
-            editLog.loadJournal(null /* GlobalStateMgr */, entity);
+            editLog.loadJournal(GlobalStateMgr.getCurrentState(), entity);
```

Comment:
check where does the `replayTo` called, I remember it used in two places 1. leader restart, replay the journal before serving. 2. leader creates a new image in checkpoint thread.

In the second case, `GlobalStateMgr.getCurrentState()` may be not correct.

Reply:
In `editLog.loadJournal`, Only the following branch will be executed. So it is ok to pass null or GlobalStateMgr.getCurrentState().

```
                case OperationType.OP_STARMGR: {
                    StarMgrJournal j = (StarMgrJournal) journal.getData();
                    StarMgrServer.getCurrentState().getStarMgr().replay(j.getJournal());
                    break;
                }
```

In the `GlobalStateMgr.getCurrentState()`, we can safely get an instance.

To ensure safety, I think I'd better reset it to null value.

Reply:
done